tests/parm/input_global_parent.nml.IN

```diff
@@ -0,0 +1,410 @@
+&atmos_model_nml
+  blocksize = @[BLOCKSIZE]
+  chksum_debug = .false.
+  dycore_only = .false.
+  ccpp_suite = "@[CCPP_SUITE]"
+  
+/
+
+&diag_manager_nml
+  prepend_date = .false.
+  max_output_fields = @[MAX_OUTPUT_FIELDS]
+  
+/
+
+&fms_nml
+  clock_grain = 'ROUTINE'
+  domains_stack_size = @[DOMAINS_STACK_SIZE]
+  print_memory_usage = .false.
+  
+/
+
+&fms2_io_nml
+  netcdf_default_format = "netcdf4"
+/
+
+&fv_core_nml
+  layout = @[INPES],@[JNPES]
+  io_layout = 1,1
+  npx = @[NPX]
+  npy = @[NPY]
+  ntiles = @[NTILES]
+  npz = @[NPZ]
+  dz_min =  @[DZ_MIN]
+  grid_type = 0
+  make_nh = @[MAKE_NH]
+  fv_debug = .false.
+  range_warn = .true.
+  reset_eta = .false.
+  n_sponge = @[N_SPONGE]
+  n_zs_filter = @[N_ZS_FILTER]
+  nudge_qv = .false.
+  nudge_dz = .false.
+  tau = @[FV_CORE_TAU]
+  rf_cutoff = @[RF_CUTOFF]
+  rf_fast = @[RF_FAST]
+  sg_cutoff = @[SG_CUTOFF]
+  upoff = @[UPOFF]
```

Comment:
Line 45-47:  the current global workflow does not have lines 45-47 (i.e. `rf_fast, sg_cutoff, upoff`). Are they the special requirements for global nesting?

Reply:
No. These are required for the global_nest_v1 physics suite.

Reply:
If they're not set correctly in global-workflow, then that is a bug in global-workflow that must be corrected.

Reply:
Thanks,  I will check with Anders and Lisa about this as the example input.nml provided by Lisa and Evelyn did not have these three lines.

Reply:
The correct values for some of those variables depend on resolution. It is possible the resolution they used had the namelist default values.

Reply:
parsing_namelists_FV3.sh does not have those 3 variables. So I assume the default values work for all resolutions. 

If we don't need to set special values for those 3 variables, I suggest we remove them for simplicity.

Reply:
The parsing_namelists_FV3.sh does not generate correct namelists for the global_nest_v1 physics suite.

Reply:
What values do we need to set for `rf_fast, sg_curff, upoff`?

Reply:
We're doing experiments to determine how best to configure the global_nest_v1 physics.

The best known working values, for now, are in the HAFS-AR repository. (Except full_zs_filter, which is erroneously set to true right now.)

Reply:
@guoqing-noaa I don't know what these values should be, and I don't see them in the global-workflow either. I only find them related to the dycore inside the model:

atmos_cubed_sphere/docs/doc_source/fv3_technical_2021.tex:Optionally, there is a ``fast Rayleigh friction'' which can be applied on each acoustic timestep, after the winds are updated with the pressure-gradient force. This is more rapidly applied (although on the same damping timescale) and thus can be more stable, although it does not yet support energy conservation. This can be enabled with the \texttt{rf_fast} namelist option

atmos_cubed_sphere/model/fv_arrays.F90:   real    :: sg_cutoff = -1   !< cutoff level for fv_sg_adj (2dz filter; overrides n_sponge)

upoff seem to be related to the dycore nest, and not used in the global configuration. 

@SamuelTrahanNOAA can you give an explanation why you introduced them, or if they were part of the HAFS nest configuration we started from?

Reply:
I read through the code how upoff is used.

There is a "border region" in the nest which is not fed back to the parent during upscale feedback. The larger this region, the less of an effect the nest has on its own boundary conditions. Setting upoff to 2 doubles the width of that region.